parse_lib.py

```diff
@@ -106,6 +106,12 @@ def update_anchor_href(anchor):
         anchor['target'] = '_blank'
 
 
+def convert_to_img(html, equation):
```

Comment:
RE function name, there definitely could be objects that aren't function names-- at the moment though, there's no application logic for deciphering between what is or isn't an SVG since there wasn't a need to make that distinction during this current version of the standard.

I believe there's a field in the `object` tag or its children that explicitly has an SVG type attribute on it, so I'll search for that in the initial search on line 95 and then use the name `convert_svg_object_to_img`. That seems like the most future-proof option